src/presenters/open-graph/image-presenter.php

```diff
@@ -94,16 +94,54 @@ public function get() {
 	 */
 	protected function filter( $image ) {
 		/**
-		 * Filter: 'wpseo_opengraph_image' - Allow changing the Open Graph image.
+		 * Filter: 'wpseo_opengraph_image' - Allow changing the Open Graph image url.
 		 *
 		 * @param string                 $image_url    The URL of the Open Graph image.
 		 * @param Indexable_Presentation $presentation The presentation of an indexable.
 		 */
 		$image_url = \trim( \apply_filters( 'wpseo_opengraph_image', $image['url'], $this->presentation ) );
-		if ( ! empty( $image_url ) && \is_string( $image_url ) ) {
+		if ( ! empty( $image_url ) ) {
 			$image['url'] = $image_url;
 		}
 
+		if ( isset( $image['type'] ) ) {
+			/**
+			 * Filter: 'wpseo_opengraph_image_type' - Allow changing the Open Graph image type.
+			 *
+			 * @param string                 $image_url    The type of the Open Graph image.
```

Comment:
That was probably copy paste from the above filter, right? We need the new parameter to be described here.

Reply:
Same for the below new filters, as well :)

Reply:
@thijsoo I dont see this being addressed?

Reply:
Apologies, I did a very poor job of explaining the second time.

I think that we have to have `int` params for the width and the height. Considering we're casting them into integers later anyway.

But I'm maybe wrong? 🤔